src/index.tsx

```diff
@@ -20,4 +17,4 @@ const Root = () => (
   </Provider>
 );
 
-root.render(<Root />);
+ReactDOM.render(<Root />, document.getElementById('root'));
```

Comment:
use recommended method:
![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/86923089/6f6a3830-a1fd-4779-984a-83f577790194)

https://react.dev/reference/react-dom/client/createRoot

ask in the chat if you need help